components/global/core/io.cellery.observability.api/pom.xml

```diff
@@ -72,15 +73,47 @@
             <groupId>org.apache.httpcomponents</groupId>
             <artifactId>httpmime</artifactId>
         </dependency>
+        <dependency>
+            <groupId>org.apache.httpcomponents</groupId>
+            <artifactId>httpcore</artifactId>
+        </dependency>
         <dependency>
             <groupId>org.json</groupId>
             <artifactId>json</artifactId>
         </dependency>
-
+        <dependency>
```

Comment:
Please remove the dependencies that are not directly used in this module. I believe you only need to add them in the docker POMs